### ISSUE TITLE
describe_container_instances argument is insufficient

### ISSUE THREAD

**Issue by matsuzj** [k1LoW/awspec#228]
Hi @k1LoW!
I always use awspec!

When '--cluster' is not specified.
containerInstances is nothing.
```
➜  ~ aws ecs describe-container-instances --container-instances 11111111-2222-3333-4444-555555555555
{
    "failures": [
        {
            "reason": "MISSING",
            "arn": "arn:aws:ecs:ap-northeast-1:000000000000:container-instance/11111111-2222-3333-4444-555555555555"
        }
    ],
    "containerInstances": []
}
```

https://github.com/matsuzj/awspec/blob/master/spec/type/ecs_container_instance_spec.rb#L5-L14
Can you only set container-instance ID?
```
describe ecs_container_instance('my-container-instance') do
  it { should exist }
  it { should be_active }
  it { should_not be_inactive }
  its(:status) { should eq 'ACTIVE' }
  its(:container_instance_arn) do
    should eq 'arn:aws:ecs:us-east-1:123456789012:container-instance'\
              '/f2756532-8f13-4d53-87c9-aed50dc94cd7'
  end
end
```

https://github.com/matsuzj/awspec/blob/master/lib/awspec/helper/finder/ecs.rb#L21-L24
Maybe, There are two arguments.
```
def find_ecs_container_instances(cluster_name, container_instances)
  res = ecs_client.describe_container_instances(cluster: cluster_name, container_instances: container_instances)
  res.container_instances if res.container_instances
end
```

**Comment by k1LoW** [comment]
Hi @matsuzj . Thank you for your report!

I fixed #233 .

`describe_container_instances` need 2 args.
If  ecs_container_instance "only set container-instance ID", awspec will have to search all clusters and instances. ( [list_clusters](http://docs.aws.amazon.com/sdkforruby/api/Aws/ECS/Client.html#list_clusters-instance_method) -> [describe_container_instances](http://docs.aws.amazon.com/sdkforruby/api/Aws/ECS/Client.html#describe_container_instances-instance_method) )

So, I adopted `cluster` option.

How about this idea?

**Comment by matsuzj** [comment]
Hi @k1LoW.
Thank you for reply.

The container_instance of resource must need 2 args.
I change branch of change-ecs_container_instance-api and tested.

LGTM!!!

Test Case
```
describe ecs_container_instance('11111111-2222-3333-4444-555555555555'), cluster: 'my-cluster' do
  its(:status) { should eq 'ACTIVE' }
  its(:agent_connected) { should eq false }
  its(:running_tasks_count) { should eq 2 }
end
```
Result
```
Failures:

  1) ecs_container_instance '11111111-2222-3333-4444-555555555555' agent_connected should eq false
     Failure/Error: its(:agent_connected) { should eq false }

       expected: false
            got: true

       (compared using ==)
     # ./spec/ecs_container_instance_spec.rb:5:in `block (2 levels) in <top (required)>'

  2) ecs_container_instance '11111111-2222-3333-4444-555555555555' running_tasks_count should eq 2
     Failure/Error: its(:running_tasks_count) { should eq 2 }

       expected: 2
            got: 1

       (compared using ==)
     # ./spec/ecs_container_instance_spec.rb:6:in `block (2 levels) in <top (required)>'

Finished in 0.19587 seconds (files took 0.96627 seconds to load)
3 examples, 2 failures
```

**Comment by k1LoW** [comment]
@matsuzj Thank you!

**Comment by k1LoW** [comment]
merged #233
